src/actions/actions.factory.js

```diff
@@ -34,7 +35,7 @@ module.exports = (
       case SKIP:
         return skip(parsedCommand, usernameToIds, myUserID, events);
       case STATUS:
-        return { type: STATUS };
+        return status(parsedCommand);
```

Comment:
`status` is not really an `action`. The state is independent of the status, I think we should remove status from actions and add some special handling for it either in parser or at the request level.